Promact.Oauth.Server/src/Promact.Oauth.Server/Controllers/ProjectController.cs

```diff
@@ -449,7 +443,7 @@ public class ProjectController : Controller
         *   "error": "ProjectNotFound"
         * }
         */
-        [ServiceFilter(typeof(CustomAttribute))]
+        [Authorize("ReadProject")]
```

Comment:
To use string constant here. I need to use static variable of String Constant. Is that ok???